### PR TITLE
[doc]modify block cache

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: docs/manual-CN/1.overview/2.quick-start/2.FAQ.md [hunk 1]
```diff
@@ -161,6 +162,19 @@ nebula> SHOW CONFIGS;
...
+### 修改  RocksDB block cache
+
+更改 storage 的配置文件 `storaged.conf`（ 路径为 `/home/user/nebula/build/install/etc`，路径可能因人而异）并重启，例如：
+
+```bash
+# Change rocksdb_block_cache to 1024
+--rocksdb_block_cache = 1024
+```
+
+使用 `curl 127.0.0.1:12114/get_flags | grep rocksdb_block_cache` 查看配置是否生效。
```
Comment (wilsonyou) [inline docs/manual-CN/1.overview/2.quick-start/2.FAQ.md]: It's better to display the result if it works

where to execute the command, on command line?

Comment (whitewum) [reply docs/manual-CN/1.overview/2.quick-start/2.FAQ.md]: right

File: docs/manual-CN/1.overview/2.quick-start/2.FAQ.md [hunk 2]
```diff
@@ -161,6 +162,28 @@ nebula> SHOW CONFIGS;
 
 [[↑] 回到顶部](#常见问题)
 
+### 修改  RocksDB block cache
+
+更改 storage 的配置文件 `storaged.conf`（ 路径为 `/home/user/nebula/build/install/etc`，路径可能因人而异）并重启，例如：
```
Comment (whitewum) [inline docs/manual-CN/1.overview/2.quick-start/2.FAQ.md]: 默认是/usr/local/nebula/etc/ 尽量用默认的

File: docs/manual-CN/1.overview/2.quick-start/2.FAQ.md [hunk 3]
```diff
@@ -161,6 +162,28 @@ nebula> SHOW CONFIGS;
...
+
+```bash
+# Change rocksdb_block_cache to 1024
```
Comment (whitewum) [inline docs/manual-CN/1.overview/2.quick-start/2.FAQ.md]: 1024MB

File: docs/manual-CN/1.overview/2.quick-start/2.FAQ.md [hunk 4]
```diff
@@ -161,6 +162,28 @@ nebula> SHOW CONFIGS;
...
+--rocksdb_block_cache = 1024
+```
```
Comment (whitewum) [inline docs/manual-CN/1.overview/2.quick-start/2.FAQ.md]: 然后重启storaged： /usr/local/nebula/scripts/nebula.storage restart -- check the command again ( I'm not pretty sure)

File: docs/manual-CN/1.overview/2.quick-start/2.FAQ.md [hunk 5]
```diff
@@ -161,6 +162,28 @@ nebula> SHOW CONFIGS;
...
+
+在终端运行 `curl 127.0.0.1:12114/get_flags | grep rocksdb_block_cache` 查看配置是否生效。
```
Comment (whitewum) [inline docs/manual-CN/1.overview/2.quick-start/2.FAQ.md]: by default , the port I think is 12000

**Pull request merged**